File: cslavb/Csla/Security/ICheckRoles.vb
Namespace Security

    ''' <summary>
    ''' Interface defining an object that
    ''' checks IsInRole.
    ''' </summary>
    ''' <remarks></remarks>
    Public Interface ICheckRoles
        ''' <summary>
        ''' Returns a value indicating whether the current
        ''' user is in the specified security role.
        ''' </summary>
        ''' <param name="role">
        ''' Role to check.
        ''' </param>
        Function IsInRole(ByVal role As String) As Boolean

    End Interface
End Namespace

File: cslacs/Csla/Security/ICheckRoles.cs
using System;

namespace Csla.Security
{
  /// <summary>
  /// Interface defining an object that
  /// checks IsInRole.
  /// </summary>
  public interface ICheckRoles
  {
    /// <summary>
    /// Returns a value indicating whether the current
    /// user is in the specified security role.
    /// </summary>
    /// <param name="role">
    /// Role to check.
    /// </param>
    bool IsInRole(string role);
  }
}
